domains/Accounts/Controllers/AccountsProfileController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Accounts\Resources\UserResource;
+use Illuminate\Http\Response;
+
+class AccountsProfileController extends Controller
+{
+    public function __invoke(): Response
+    {
+        return new Response(
+            [
+                'data' => UserResource::make(auth()->user())
+            ],
+            Response::HTTP_OK);
+    }
```

Comment:
We can simply return the resource class, as it's being done on other similar controllers

Reply:
Okay, I'm going to change.
but I need to send a new Response(), because is a POST request by default it sends 201 code. 

I don't think it makes much sense.

Changed to:
`return new Response(UserResource::make(Auth::user()), Response::HTTP_OK);`